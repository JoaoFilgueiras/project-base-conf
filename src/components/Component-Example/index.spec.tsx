import { render, screen } from '@testing-library/react'

import ComponentExample from '.'

describe('<ComponentExample/>', () => {
  it('should render the headind', () => {
    const { container } = render(<ComponentExample />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
