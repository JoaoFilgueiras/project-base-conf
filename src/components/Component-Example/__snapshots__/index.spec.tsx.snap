// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ComponentExample/> should render the headind 1`] = `
<main>
  <h1>
     React Avan√ßado 
  </h1>
</main>
`
